src/Illuminate/Translation/Translator.php

```diff
@@ -184,7 +185,7 @@ protected function sortReplacements(array $replace)
      * Get a translation according to an integer value.
      *
      * @param  string  $key
-     * @param  int     $number
```

Comment:
There was a `use` statement for Countable. So I removed trailing slash. Returned it back.